### PR TITLE
fix(git): resync 21.04.x to dev-21.04.x

### PR DESCRIPTION
**Pull request by chgautier**
## Description

resync 21.04.x to dev-21.04.x

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [x] 21.04.x
- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x (master)

### CODE REVIEW

**Review by a-launois — CHANGES_REQUESTED** [review]
There is unresolved conflict in some files  (ex: .github/CODEOWNERS, .github/CODEOWNERS)

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,12 +1,22 @@
+<<<<<<< HEAD
```
Comment (kduret) [inline .github/CODEOWNERS]: conflict not handled

**Review by kduret — CHANGES_REQUESTED** [review]

**Review by jdelpierre — APPROVED** [review]

File: www/class/centreonAuth.SSO.class.php [hunk 2]
```diff
@@ -45,6 +45,9 @@ class CentreonAuthSSO extends CentreonAuth
     private const START = 0;
     private const LENGTH = 8;
 
+    private const START = 0;
+    private const LENGTH = 8;
+
```
Comment (callapa) [inline www/class/centreonAuth.SSO.class.php]: Already defined
```suggestion
```

**Review by callapa — CHANGES_REQUESTED** [review]

**Review by callapa — APPROVED** [review]

**Comment by chgautier** [comment]
Will remake a new synchro branch

**Pull request closed without merging**